plugins/modules/mso_schema_site_anp_epg_staticport.py

```diff
@@ -85,6 +86,11 @@
     - C(untagged) means B(Access (untagged)).
     type: str
     choices: [ native, regular, untagged ]
+    default: untagged
+  micro_segment_vlan:
+    description:
+    - Primary micro-seg VLAN of dpc
```

Comment:
Primary micro-seg VLAN is for all 3 so we should be clear that it is for Port, Direct Port Channel (dpc) and Virtual Port Channel (vpc).